runner/runners/polling.go

```diff
@@ -3,21 +3,31 @@ package runners
 import (
 	"time"
 
+	"github.com/davecgh/go-spew/spew"
 	"github.com/scootdev/scoot/runner"
+	"log"
 )
 
 // polling.go: turns a StatusQueryNower into a StatusQuerier by polling
 
 // NewPollingStatusQuerier creates a new StatusQuerier by polling a StatusQueryNower that polls every period
 func NewPollingStatusQuerier(del runner.StatusQueryNower, period time.Duration) *PollingStatusQuerier {
-	return &PollingStatusQuerier{del, period}
+	runner := &PollingStatusQuerier{del, period}
+	log.Println("************** runner definition")
+	runnerDesc := spew.Sdump(runner)
+	log.Println(runnerDesc)
+	return runner
 }
 
 // NewPollingService creates a new Service from a Controller, a StatusEraser, and a StatusQueryNower.
 // (This is a convenience function over NewPollingStatusQuerier
 func NewPollingService(c runner.Controller, e runner.StatusEraser, nower runner.StatusQueryNower, period time.Duration) runner.Service {
 	q := NewPollingStatusQuerier(nower, period)
-	return &Service{c, q, e}
+	runner := &Service{c, q, e}
```

Comment:
We don't need this one since it calls NewPollingStatusQuerier.

I don't think it's worth the verbosity to print this every time the function is called, especially if we can do it once for every type on startup after parsing and evaluating the config. More to the point, it looks like the repeated prints may currently be devoid of actionable info.